packages/dds/map/src/test/mocha/map.spec.ts

```diff
@@ -773,6 +777,133 @@ describe("Map", () => {
 		});
 	});
 
+	describe("Attribution", () => {
+		describe("Local State", () => {
+			let map: SharedMap;
+
+			beforeEach(async () => {
+				const options: IMapOptions = { attribution: { track: true } };
+				map = createLocalMap("testMap", options);
+			});
+
+			it("Should not generate attribution in local state", () => {
+				map.set("first", "second");
```

Comment:
key-value names are a little confusing here. i'd do a different scheme (e.g. "first" -> 1, "second" -> 2, etc or something like that)